Alignment/OfflineValidation/python/TkAlAllInOneTool/primaryVertexValidationTemplates.py

```diff
@@ -0,0 +1,374 @@
+PrimaryVertexValidationTemplate="""
+import FWCore.ParameterSet.Config as cms
+import sys
+ 
+isDA = .oO[isda]Oo.
+isMC = .oO[ismc]Oo.
+
+process = cms.Process("Demo") 
+
+###################################################################
+# Event source and run selection
+###################################################################
+.oO[datasetDefinition]Oo.
+#process.load("Alignment.OfflineValidation..oO[dataset]Oo._cff")
+
+###################################################################
+#  Runs and events
+###################################################################
+runboundary = .oO[runboundary]Oo.
+process.source.firstRun = cms.untracked.uint32(int(runboundary))
+#process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(MAXEVENTSTEMPLATE) )
+
+###################################################################
+# JSON Filtering
+###################################################################
+if isMC:
+     print ">>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is simulation!"
+     runboundary = 1
+else:
+     print ">>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is real dATA!"
+     if ('.oO[lumilist]Oo.'):
+          print ">>>>>>>>>> testPVValidation_cfg.py: msg%-i: JSON filtering with: .oO[lumilist]Oo. "
+          import FWCore.PythonUtilities.LumiList as LumiList
+          process.source.lumisToProcess = LumiList.LumiList(filename ='.oO[lumilist]Oo.').getVLuminosityBlockRange()
```

Comment:
@mmusich I think this is a bit in conflict with `.oO[datasetDefinition]Oo.` in line 13 because this also might define a JSON selection
Defining it twice does not harm, but it might confuse people which use a predefined dataset from the python folder which might include a JSON, while the user sets JSON to none.